### ISSUE TITLE
GCam 8.1. crashes when opening

### ISSUE THREAD

**Issue by geronimoge** [lukaspieper/Gcam-Services-Provider#24]
**Describe the bug**
Unofficial Lineage OS 18.1. is running. 
Gcam-Services-Provider v1.1 is installed as well as GCam 8.1.101.
As soon as I start the GCam it won't open but crash.

**To Reproduce**
1. Install Gcam-Services-Provider v1.1
2. Downoad and install GCam 8.1.101 [from here](https://www.celsoazevedo.com/files/android/google-camera/dev-bsg/f/dl5/) no matter which apk, all are crashing.
3. Open app drawer
4. Click on 'camera' to open GCam 8.1.101
5. App starts and crashes directly
6. Open app drawer again
7. Click on 'camera' to open GCam 8.1.101
8. App starts and crashes directly
9. System shows error ’Camera keeps stopping’

**Expected behavior**
I expect that the app is starting with asking for permissions, since it is the first launch.

**Log**
[logcat_03-15-2021_20-10-39.txt](https://github.com/lukaspieper/Gcam-Services-Provider/files/6143940/logcat_03-15-2021_20-10-39.txt)

**Smartphone (please complete the following information, the more detailed, the better)**
 - Device: Samsung Galaxy S9
 - OS/ROM: [Unofficial LineageOS 18.1.](https://forum.xda-developers.com/t/rom-s9-s9-unofficial-lineageos-18-1-23-02-2021.4203777/)
 - [Google Camera version](https://www.celsoazevedo.com/files/android/google-camera/dev-bsg/f/dl5/)
 - Gcam-Services-Provider version: 1.1.

**Additional context**
I just want to thank you for this great piece of work. I'm so happy that you stepped forward and decided to develop this.
It is helping a bunch of people and I hope you will continue for a long time.
So thank you very much!

**Comment by lukaspieper** [comment]
Hi,

thank you for the kind words and especially for the detailed information and the log. The error message is very similar to the one from issue #6. Did you install the app with `--force-queryable`?

**Comment by geronimoge** [comment]
Thanks for the quick feedback.
And no, I originally downloaded and installed your app directly in the phone.

Now I downloaded the latest version of your app again on the notebook.
Then I installed it in the terminal via adb and as described in #6 by innir as follows:
$adb push app-debug.apk /data/local/tmp
$ adb shell pm install --force-queryable /data/local/tmp/app-debug.apk
Now the GCam app is working again.
Thanks again!
